dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -25,12 +26,12 @@ class FileOrBufferHandler:
 
     def __init__(
         self,
-        filepath_or_buffer,
-        open_method="r",
-        encoding=None,
-        seek_offset=None,
-        seek_whence=0,
-    ):
+        filepath_or_buffer: Union[str, StringIO, BytesIO],
+        open_method: str ="r",
+        encoding: Optional[str]=None,
+        seek_offset: Optional[int]=None,
+        seek_whence: int=0,
+    ) -> None:
```

Comment:
Is `None` right given this? https://github.com/capitalone/DataProfiler/pull/682/files#diff-c58c4230bad99240e958de2037065b73ad4498c5e64f2e53b18350b0092f8df0R47

```:return: TextIOBase or BufferedIOBase class/subclass```

Reply:
This should still be `None`.